src/app/components/footer/footer.component.css

```diff
@@ -35,3 +35,10 @@ footer nav ul {
 footer nav li {
   list-style: none;
 }
+
+.footer-logotype {
```

Comment:
Can you add a small margin bottom, so the logo is separated from the bottom of the screen?
e.g. `margin-bottom: 0.5rem;`